### ISSUE TITLE
Use Go http package

### ISSUE THREAD

**Issue by bradrydzewski** [drone-plugins/drone-bintray#2]
we can use the native `net/http` package in Go to make these requests. We can change:

```
    this.execCommand("curl", bintray.ApiKey,
        "-H", fmt.Sprintf("X-Bintray-Debian-Distribution: %s", this.Distr),
        "-H", fmt.Sprintf("X-Bintray-Debian-Component: %s", this.Component),
        "-H", fmt.Sprintf("X-Bintray-Debian-Architecture: %s", strings.Join(this.Arch, ",")),
        "-H", fmt.Sprintf("X-Bintray-Override: %d", boolToInt(this.Override)),
        "-H", fmt.Sprintf("X-Bintray-Publish: %d", boolToInt(this.Publish)),
        "-T", fmt.Sprintf("%s/%s", path, this.File),
        fmt.Sprintf("-u%s:%s", bintray.Username, bintray.ApiKey),
        this.getEndpoint())
```

to something like this:

```
req, err := http.NewRequest("POST", endpoint, file)
req.Headers.Add("X-Bintray-Debian-Distribution", this.Distr)
....
```

**Comment by tooda02** [comment]
Created PR #3

**Comment by jackspirou** [comment]
Merged PR #3 fixed this issue.
